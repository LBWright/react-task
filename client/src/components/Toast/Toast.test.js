import React from 'react'
import { mount } from 'enzyme'
import renderer from 'react-test-renderer'

import Toast from '.'

describe('<Toast />', () => {
  // named to prevent namespacing confusion
  const toastComponent = mount(<Toast />)
  it('should render without crashing', () => {
    expect(toastComponent).toBeTruthy()
  })
  it('should render <ToastContainer />', () => {
    expect(toastComponent.find('ToastContainer')).toBeTruthy()
  })
  it('should match snapshot', () => {
    const toastSnap = renderer.create(<Toast />)
    expect(toastSnap.toJSON()).toMatchSnapshot()
  })
})
