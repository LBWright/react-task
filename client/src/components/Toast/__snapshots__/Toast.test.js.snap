// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toast /> should match snapshot 1`] = `
<div
  className="Toastify"
/>
`;
